packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -104,7 +104,8 @@ afterAll(async () => {
   provider.polling = false;
 });
 
-it('Create a directly funded channel between two engines ', async () => {
+// TODO: This should be re-enabled once the chain service has been decoupled from the engine
```

Comment:
This should be updated and re-enabled when https://github.com/statechannels/statechannels/issues/3548 is completed